src/bootstrap-table.js

```diff
@@ -1862,11 +1862,12 @@ class BootstrapTable {
 
   trigger (_name, ...args) {
     const name = `${_name}.bs.table`
+    args[args.length] = this
     this.options[BootstrapTable.EVENTS[name]](...args)
-    this.$el.trigger($.Event(name), args)
+    this.$el.trigger($.Event(name, { sender: this }), args)
 
     this.options.onAll(name, args)
```

Comment:
`this.options.onAll(name, [...args, this])`

Reply:
Thanks, done.